app/workers/expire_rent_email_worker.rb

```diff
@@ -0,0 +1,8 @@
+class ExpireRentEmailWorker
+  include Sidekiq::Worker
+
+  def perform
+    rents = Rent.where(end_date: Time.zone.today)
+    rents.map { |rent| RentMailer.expire_rent_send(rent.id).deliver_now }
```

Comment:
Also, any reason to use `delivery_now` instead  of `deliver_later`?

Reply:
was thinking why not now ? If you think is better later I can doit like that